### DISCUSSION TITLE
Why I don't have Breadcrumbs in yaml file?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [nvimdev/lspsaga.nvim#1218] [Q&A] [answered]
![image](https://github.com/nvimdev/lspsaga.nvim/assets/95092244/a811705c-3163-48ec-b0fd-2711cd72aa93)

Does lspsaga has Breadcrumbs for yaml file?

**Comment by glepnir** [comment, marked as answer]
did you config a server for yaml?

**Reply by LintaoAmons** [reply]
Oh, yes!
